sycl/test-e2e/Basic/context.cpp

```diff
@@ -67,19 +67,5 @@ int main() {
                            sycl::property_list{});
     sycl::context Context7(std::vector<sycl::device>{deviceA},
                            sycl::property_list{});
-    sycl::context Context8(
```

Comment:
This seems rather unrelated to me. In any case, if we want to drop testing for this deprecated feature, I think we can do that as a separate PR together with tests to make sure that we are not dropping test coverage without dropping the implementation.

Reply:
I think this change was reverted, the PR no longer touches this file.